openedx/core/djangoapps/user_api/legacy_urls.py

```diff
@@ -39,6 +39,8 @@
     urlpatterns += [
         url(r'^v1/account/login_session/$', user_api_views.LoginSessionView.as_view(),
             name="user_api_login_session"),
+        url(r'^v1/account/logout_session/$', user_api_views.LogoutSessionView.as_view(),
```

Comment:
1. Why didn't this exist before?
2. Can we eventually rename legacy_urls.py? :)  To me, 'legacy' doesn't mean what was intended here.